libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -30,8 +30,8 @@ export const MAX_OUT_OF_DATE_MONTHS = 6
 const LOG_CATEGORY = 'userprofile-service'
 
 // eslint-disable-next-line
-const handleError = (error: any) => {
-  logger.error('Userprofile error', {
+const handleError = (error: any, details?: string) => {
+  logger.error(details || 'Userprofile error', {
     error: JSON.stringify(error),
     category: LOG_CATEGORY,
   })
```

Comment:
```suggestion
  logger.error(details || 'Userprofile error', error)
```

Reply:
But then we will not be able to filter the logs in DD by category. In this case I can go in, and filter with `@category:userprofile-service` to see all the errors coming from this service, which has proven to be a nice debugging tool.